src/index.ts

```diff
@@ -31,10 +45,31 @@ export default class implements ConnectPlugin {
        return { description, snippet, lang };
     }
 
+    /** 
+     * Note: If we eventually want to support aliases from non-Vue CLI projects,
+     * we'll likely want to add a test for this and other methods to retrieve those aliases
+     * For now, it's not worth the effort to setup & maintain an entire dummy Vue CLI project,
+     * with the relevant vue.config.js
+     */
```

Comment:
Intentionally skipped tests for this case, for now. The proper test would be to initialize a sample Vue CLI project within this repo, and have it import at least one aliased file. Given that the failure is graceful, the overhead of maintaining a CLI project in this repo does not yet seem worth it, though it can certainly be done at some point.